css/style.css

```diff
@@ -104,3 +121,530 @@ body {
   display: flex;
   padding-left: 67px;
 }
+
+.cards {
+  width: 375px;
+  height: 4120px;
+  background-color: #e5e5e5;
+  margin: auto;
+  display: flex;
+  flex-direction: column;
+}
+
+.first_card {
+  width: 327px;
+  height: 552px;
+  border: 1px solid #dfe1e6;
+  background-color: #fff;
+  border-radius: 16px;
+  margin: 116px auto 0;
+}
+
+.first_card img {
+  padding-top: 16px;
+  width: 295px;
+  height: 220px;
+  margin-left: 16px;
+}
+
+.first_card h2 {
+  width: 295px;
+  height: 36px;
+  margin-left: 16px;
+  font-family: 'poppins', sans-serif;
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 44px;
+  color: #172b4d;
+  margin-top: 8px;
+}
+
+.first_card h3 {
+  width: 55px;
+  height: 16px;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  font-family: 'poppins', sans-serif;
+  color: #344563;
+  margin-left: 16px;
+  margin-top: -8px;
+}
+
+.first_card .birth {
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+  display: flex;
+  justify-content: space-evenly;
+  margin-left: -12px;
+  margin-top: -27px;
+}
+
+.first_card .year {
+  margin-left: -45px;
+}
+
+.first_card p {
+  width: 295px;
+  height: 76px;
+  font-family: 'poppins1', sans-serif;
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 24px;
+  color: #344563;
+  margin-left: 16px;
+}
+
+.first_card button {
```

Comment:
- [ ] I kindly suggest that you add the button interactions for hover as shown in the Figma template below:

![Screenshot from 2022-04-27 19-12-50](https://user-images.githubusercontent.com/52934128/165592912-7d8bf224-8e6b-467d-bb19-cf63cd2b40fd.png)

the button on hover has the `background-color: #6070FF;` 

### EXAMPLE:
```
**for hover state**
```
button:hover {
    background-color: #6070FF;
}
```